src/fragments/lib/push-notifications/ios/getting_started/60_init_push_notifications.mdx

```diff
@@ -0,0 +1,64 @@
+To initialize the Amplify Push Notifications, use the `Amplify.addPlugin()` method to add the AWSPinpointPushNotificationsPlugin. Next, finish configuring Amplify by calling `Amplify.configure()`.
```

Comment:
do you usually provide guidance on where to place the configure function for Swift? Should this be made clearer?

Reply:
it goes into more detail on the following lines.  Re-phrased to make it more clear that this is summarizing what you're gong to do in the section.